kratos/python_scripts/run_tests.py

```diff
@@ -197,7 +198,8 @@ def main():
                 'applications=',
                 'verbose=',
                 'level=',
-                'command='
+                'command=',
+                'using-mpi'
```

Comment:
do we need the `--` in front here?

Reply:
hm I don't think so, but I have to check again, it has been some time since I did this

Reply:
I just checked again with the new tests in the MappingApp, it works nicely as is